src/cmds/openapi/index.ts

```diff
@@ -1,4 +1,4 @@
-import type { CommandOptions } from '../lib/baseCommand';
+import type { CommandOptions } from '../../lib/baseCommand';
```

Comment:
Moved this into `src/cmds/openapi/index.ts` because the `openapi` command group is a family now. :corona: :vindiesel:

Reply:
oh the v8[^1] release notes are definitely going to be F&F-themed now

[^1]: also v8... multitudes!!!!